ts/src/bitget.ts

```diff
@@ -279,6 +279,7 @@ export default class bitget extends Exchange {
                             'order/batch-orders': 2,
                             'order/cancel-order': 2,
                             'order/cancel-batch-orders': 2,
+                            'order/modifyOrder': 2, // 10 times/1s (UID) => 20/10 = 2
```

Comment:
@sc0Vu can we update `editOrder` with this one?

Reply:
@carlosmiei yes, I'll submit another PR for this!